packages/elements/src/components/ino-accordion/ino-accordion.tsx

```diff
@@ -19,15 +20,17 @@ import classNames from 'classnames';
 })
 export class Accordion implements ComponentInterface {
   @Element() el: HTMLInoAccordionElement;
+  @State() contentId: string;
```

Comment:
Instead of introducing a state variable that triggers an extra rerender after it has initially been loaded, I would suggest just using a private variable that is initialized on creation. Like this:

```ts
private readonly contentId = `ino-accordion-content-${this.el.id}`
```

This variable can then be used in the `render` method without triggering an extra rendering process.